### ISSUE TITLE
Doesn't work properly on Windows

### ISSUE THREAD

**Issue by littledumb** [zedgu/ovenware#1]
The problem is mainly due to the use of regular expression to replace part of the file path. In loader.js:

```
paths[file.replace(new RegExp('^' + path.resolve(root) + '/'), '').replace(/.js$/, '').toLowerCase()] = file;
```

**Comment by zedgu** [comment]
I have no Windows PC for now, can u fix this problem for me?
or I'll check this problem later.

**Comment by zedgu** [comment]
Please check v0.1.3.
It should be ok for windows i guess

**Comment by littledumb** [comment]
Thanks! I'll tell my colleague (I don't have Windows PC either :) ).

**Comment by zedgu** [comment]
I just published v0.2.0，it fixed windows issues i think.

**Comment by punmechanic** [comment]
Issues are resolved on Win7 x64
